index.bs

```diff
@@ -359,7 +359,7 @@ NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still
 
   It is possible for a file compliant to this [=AV1 Image File Format=] to not be able to declare an AVIF profile, if the corresponding AV1 encoding characteristics do not match any of the defined profiles.
 
-  NOTE: [[!AV1]] supports 3 bit depths: 8, 10 and 12 bits, and the maximum dimensions of a coded image is 65536x65536, when coded with level 31.
+  NOTE: [[!AV1]] supports 3 bit depths: 8, 10 and 12 bits, and the maximum dimensions of a coded image is 65536x65536, when coded with level index 31 (maximum parameters).
```

Comment:
Thank you. If we refer to the table in section A.3 of the AV1 spec, when `seq_level_idx` is 31, the 'level' column says "maximum parameters", so overall I see why you want to avoid using "level 31", but the term "level index" is never mentioned either in the AV1 specification. Could we just say:
> 65536x65536, when `seq_level_idx` is set to 31 (maximum parameters level).

Reply:
Thank you, I agree with your comment and have made the suggested change.